math/int.go

```diff
@@ -474,3 +474,28 @@ func FormatInt(v string) (string, error) {
 
 	return sign + v, nil
 }
+
+// ParseInt formats an value-rendered string into an integer
+func ParseInt(v string) (Int, error) {
```

Comment:
Hey @atheeshp, after talking with Aaron, let's put the 3 functions `ParseInt`, `ParseDec` and `ParseCoins` inside `tx/textual/valuerenderer` for now, as private functions.

We still need to decide how autocli does parsing. Then, we can extract the relevant parts

Reply:
up here too